src/com/minseung/datastructure/LinkedListNode.java

```diff
@@ -279,6 +290,33 @@ public LinkedList getSum(LinkedList l1, LinkedList l2) {
             }
             l3.append(result);
         }
+
+        if (carry != 0) {
+            l3.append(carry);
+        }
+
         return l3;
     }
+
+    public LinkedList.Node getSumUsingRecursive(LinkedList.Node n1, LinkedList.Node n2, int carry) {
```

Comment:
Or, you can create a child class from this class, and implement getSum differently.

+ 1